internal/cmd/kafka/command_consumergroup_lag_summarize.go

```diff
@@ -48,14 +47,19 @@ func (c *lagCommand) newSummarizeCommand() *cobra.Command {
 func (c *lagCommand) summarize(cmd *cobra.Command, args []string) error {
 	consumerGroupId := args[0]
 
-	kafkaREST, lkc, err := getKafkaRestProxyAndLkcId(c.AuthenticatedCLICommand)
+	kafkaREST, err := c.GetKafkaREST()
 	if err != nil {
 		return err
 	}
 
-	summary, httpResp, err := kafkaREST.CloudClient.GetKafkaConsumerGroupLagSummary(lkc, consumerGroupId)
+	cluster, err := c.Context.GetKafkaClusterForCommand()
 	if err != nil {
-		return kafkarest.NewError(kafkaREST.CloudClient.GetUrl(), err, httpResp)
+		return err
+	}
+
+	summary, err := kafkaREST.CloudClient.GetKafkaConsumerGroupLagSummary(cluster.ID, consumerGroupId)
```

Comment:
Does **every** Kafka REST function require a cluster ID? If so, should we add it to the Kafka REST client struct so we don't need to pass it here?

Reply:
Good call. It looks like that's the case.

Also, `GetKafkaREST` already calls `GetKafkaClusterForCommand` so we'll be able to remove a redundant call from many commands and make them a bit more efficient.